packages/components/src/autocomplete/index.js

```diff
@@ -23,7 +23,6 @@ import { getRectangleFromRange } from '@wordpress/dom';
 /**
```

Comment:
I feel this component is not well placed in the components package as if I remember properly it's written to be RichText format specific. Something to address later maybe.

Reply:
I feel the same way... This was done a long time ago though. Perhaps we can move it in a separate PR.